src/components/metadata-modal/metadata-modal.js

```diff
@@ -89,6 +94,16 @@ const MetadataModal = ({ metadata, onToggle, visible }) => {
           </button>
         </div>
       )}
+      {hasJSON && (
+        <div className="pipeline-metadata-modal__preview-json">
+          <JSONObject
+            value={JSON.parse(metadata.preview)}
+            theme={theme}
+            style={{ background: 'transparent', fontSize: '15px' }}
```

Comment:
is it fine to use inline style or should be fallback to className ?

Reply:
@jitu5 I think this is fine right?